config/initializers/setup_mail.rb

```diff
@@ -0,0 +1,11 @@
+ActionMailer::Base.smtp_settings = {
+    :address              => "smtp.gmail.com",
+    :port                 => 587,
+    :domain               => "mail.google.com",
+    :user_name            => "minahil.mustafa@devsinc.com",
+    :password             => "minahil3955",
+    :authentication       => "plain",
+    :enable_starttls_auto => true
+  }
+  
```

Comment:
Do you know about environment variables?

Reply:
Env variable describes our environment or to do some configurations on specific environment.

Reply:
This configuration is not needed for the time being but may be in future.